### DISCUSSION TITLE
Temporary library

### DISCUSSION THREAD

**Discussion by pabloto** [codefori/vscode-ibmi#1147] [General]
Hi Could the temporary library stay into an IASP?

this is my connection setting:

<img width="916" alt="image" src="https://user-images.githubusercontent.com/9719313/226413065-a16bdc4e-ecaa-4534-9cc6-d4e5f3487340.png">

VSCODE library stay on my IASP, but at login is I try to install the the CL prompt at login in codeforioutput I see:

<img width="1718" alt="image" src="https://user-images.githubusercontent.com/9719313/226413581-59176147-0d0e-4330-b130-e86a368d5cf9.png">

My iasp is ignored.

Many thanks

**Comment by pabloto** [comment]
Always look at my setting I didn't have set the IASP name, because as I can read:

If source files live within a specific ASP, please specify it here. Leave blank otherwise. You can ignore this if you have access to QSYS2.ASP_INFO as Code for IBM i will fetch ASP information automatically.

But putting the name of my iasp into this field I finally can install CL prompting.

Do I have to left the IASP name set?

Many thanks

**Comment by pabloto** [comment]
Another thing, I have 5352 object in vscode, is it normal?

**Reply by worksofliam** [reply]
You should `DLTLIB *FILE` on that library. Turn on the auto cleanup settings also.
